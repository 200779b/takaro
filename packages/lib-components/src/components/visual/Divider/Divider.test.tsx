import { Divider } from '.';
import { render } from 'test-utils';

it('Should render <Accordion />', () => {
  const { container } = render(<Divider />);
  expect(container).toMatchSnapshot();
});
