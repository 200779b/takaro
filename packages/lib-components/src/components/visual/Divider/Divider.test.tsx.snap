// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render <Accordion /> 1`] = `
.c0 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 70%;
  margin: 1rem auto 1rem auto;
}

.c1 {
  width: 100%;
  height: 1px;
  background-color: #d3d3d3;
  z-index: 0;
}

<div>
  <div
    class="c0"
    spacing="medium"
  >
    <div
      class="c1"
    />
  </div>
</div>
`;
